Woof/WoofTests/Model/Preferences/PreferencesHandlerLoadTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+final class PreferencesHandlerLoadTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        PreferencesHandlerTestHelper.cleanStorage()
+    }
+
+    override func tearDown() {
+        PreferencesHandlerTestHelper.cleanStorage()
+        super.tearDown()
+    }
+
+    func testLoadSelectedRoleExistInAPI() {
+        _ = PreferencesHandler.loadSelectedRole()
+    }
+
+    func testLoadSelectedRoleReturnsExpectedRole() {
+        _ = PreferencesHandler.saveSelectedRole(.owner)
+
+        XCTAssertEqual(PreferencesHandler.loadSelectedRole(), .owner)
+    }
+
+    func testLoadSelectedRoleReturnsDefaultCaseForNonExistingKey() {
+        XCTAssertEqual(PreferencesHandler.loadSelectedRole(), .none)
+    }
+
+    func testLoadSelectedRoleReturnsExpectedRoleWhenRewrittenByNonDefaultCase() {
+        let rewrittenRole: Role = .sitter
```

Comment:
nit:

I suggest to use given-when-then markers.

Reply:
ok, i'll add them